### ISSUE TITLE
 nemo-media-columns: enable specific column for all subfolder

### ISSUE THREAD

**Issue by anishmittal2020** [linuxmint/nemo-extensions#443]
For `$HOME/Music` I would like to enable bitrate, length for all subfolder. How can I do it?

**Comment by smurphos** [comment]
I don't think this can be done elegantly. If column preferences were part of a folder's metadata then it could be (maybe a feature request?), but as it stands column preferences are a nemo setting stored in dconf that applies to all directories. The only solution i can think off is a background script/service watching the nemo windows / process in the background and changing dconf on the fly depending on the directory being looked at - and that would fall down with multiple windows / dual pane view etc...
